### DISCUSSION TITLE
check video playing

### DISCUSSION THREAD

**Discussion by AffanAhmedUsmani** [PaulleDemon/tkVideoPlayer#38] [Q&A] [answered]
there should be a function that return true or false that a video is playing or not also there should be a soung toogle function

**Comment by PaulleDemon** [comment, marked as answer]
There is a a function that tells you if the video is paused its `tkvideoplayer.is_paused()` you can check if the video has stopped using `tkvideoplayer._stop` which holds a boolean value. As for sound check the issue https://github.com/PaulleDemon/tkVideoPlayer/issues/2
